letsencrypt/client/CONFIG.py

```diff
@@ -75,6 +75,9 @@
 NONCE_SIZE = 16
 """byte size of Nonce"""
 
+PORT = 443
```

Comment:
There are a lot of different ports, and hence the name is a bit ambiguous to me. Maybe `STANDALONE_CHALLENGE_PORT`?